dependency-versions/pom.xml

```diff
@@ -124,6 +124,7 @@
         <netty.vespa.version>4.1.107.Final</netty.vespa.version>
         <netty-tcnative.vespa.version>2.0.65.Final</netty-tcnative.vespa.version>
         <onnxruntime.vespa.version>1.17.1</onnxruntime.vespa.version>
+        <java-llama.vespa.version>2.3.5</java-llama.vespa.version>
```

Comment:
Assuming this should not be exposed to customers, please move the property to `parent/pom.xml`

Reply:
it looks to me like it belongs in this file, under
_Dependencies used internally in Vespa, not visible for users_